src/JabbR-Core/Startup.cs

```diff
@@ -43,6 +45,16 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddMvc();
             services.AddSignalR();
+
+            // Establish default settings from appsettings.json
+            services.Configure<ApplicationSettings>(Configuration.GetSection("ApplicationSettings"));
```

Comment:
Which one works? Second one that hard codes values isn't required if reading from ApplicationSettings is ok